app/controllers/provider/admin/backend_apis/base_controller.rb

```diff
@@ -11,4 +12,8 @@ class Provider::Admin::BackendApis::BaseController < Provider::Admin::BaseContro
   def find_backend_api
     @backend_api = current_account.backend_apis.accessible.find(params[:backend_api_id])
   end
+
+  def authorize_plans!
+    authorize! :manage, :plans
```

Comment:
Why not `authorhize! :manage, BackendAPI` like https://github.com/3scale/porta/blob/455e9aae0f9a27aa8b97e0bdbf72b26451901e2a/app/controllers/admin/api/backend_apis_controller.rb#L132

Reply:
I will be. Please don't review this just yet.